deploy/cr.yaml

```diff
@@ -243,3 +243,10 @@ spec:
 #       - --target="2021-06-09 14:15:11-04"
 #       restore iindividual databases
 #       - --db-include=hippo
+
+  pmm:
+    enabled: false
+    image: percona/pmm-client:2.32.0
+    imagePullPolicy: IfNotPresent
```

Comment:
We should not have it uncommented by default. Please comment it

Reply:
https://github.com/percona/percona-postgresql-operator/pull/313/commits/422611dd74c87af724c7bf38f628e6c5286f291a